NAMESPACE

```diff
@@ -1,7 +1,12 @@
 # Generated by roxygen2: do not edit by hand
 
 export(ORFScores)
+export(ORFStartCodons)
```

Comment:
This one is going maybe overboard a bit with making package user friendly, tiling and subtracting first or last 3bp should be fairly easy for anyone.